taxi/views.py

```diff
@@ -56,9 +79,25 @@ class ManufacturerDeleteView(LoginRequiredMixin, generic.DeleteView):
 
 class CarListView(LoginRequiredMixin, generic.ListView):
     model = Car
-    paginate_by = 5
+    paginate_by = 1
```

Comment:
It does not make much sense to reduce the number of elements that are displayed on a page to 1, leave this attribute with a greater value in the future

Reply:
i know but i had few elements for pagination in 5 elements on the page